client/src/components/DashboardDropdown/DashboardDropdown.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../scssStyles/mixins';
+
+.dashboard-dropdown-bg {
+  top: 0;
+  height: 100%;
+
+  width: fit-content;
+}
+
+.dashboard-dropdown-bg-display {
+  opacity: 1;
+}
+
+.dashboard-dropdown-bg-hide {
+  pointer-events: none;
+  opacity: 0;
+}
+
+.dashboard-dropdown-container {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  border-radius: 6px;
+  width: max(130px, 13%);
+  margin: auto;
+  z-index: 90;
+  border: 2px solid var(--purple-shades-muted);
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+  opacity: 0;
+  transition: opacity 300ms, transform 300ms;
+  @include devices(tablet) {
+    font-size: smaller;
+    width: 26%;
+  }
+}
+.dashboard-dropdown-filler {
+  border: none;
+  background: transparent;
+}
+.dashboard-dropdown-container-display {
+  opacity: 1;
+  transition: opacity 300ms, transform 300ms;
+}
+
+.dashboard-dropdown-item-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  background-color: var(--navbar-bg);
+  color: var(--black);
+  width: 100%;
+  height: 60px;
```

Comment:
instead of using `height`, you can use padding instead to make space around the container! padding just makes sure that things fit better on the screen, and it the container will automatically "grow" depending on the contents inside
![image](https://github.com/UofT-Frosh-Orientation/orientation-website/assets/86009011/9810468a-4864-4c8d-b8ca-6e814d28f9ce)
^ this was using `padding: 8px`